### DISCUSSION TITLE
Ring Plug-in "Unbridge" Add?

### DISCUSSION THREAD

**Discussion by sjs716** [dgreif/ring#1163] [Q&A]
Can someone please help me understand how to add Ring cameras individually if "unbridged" is chosen as an option?

Thank you in advance!

**Comment by dgreif** [comment]
It should show up as an accessory which you can add in the Home app, much like when you added the "bridge" for homebridge

**Comment by sjs716** [comment]
Thank you!

Scott J Scheur

Intense LLC

512-915-2001
